### PR TITLE
Resolve local bundles.

### PR DESCRIPTION
**Pull request by frankban**

### CODE REVIEW

**Comment by jujugui** [comment]
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://ci.jujugui.org:8080//job/charmrepo/46/
Test PASSed.

**Comment by jrwren** [comment]
:+1:

**Comment by fabricematrat** [comment]
:+1:

**Comment by mitechie** [comment]
One note on this we need to resolve local bundle files yamo files and also
bundle directories 4 bundles that would be in the store

File: local.go [hunk 1]
```diff
@@ -50,6 +50,14 @@ func (r *LocalRepository) Resolve(ref *charm.Reference) (*charm.URL, error) {
 	if ref.Revision != -1 {
 		return u, nil
 	}
+	if ref.Series == "bundle" {
+		if _, err = r.GetBundle(u); err != nil {
```
Comment (rogpeppe) [inline local.go]: I don't think we need to do this. We know the only possible revision and that's good enough for Resolve.

Comment (frankban) [inline local.go]: Done.

**Comment by rogpeppe** [comment]
LGTM, thanks!

**Comment by jujugui** [comment]
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://ci.jujugui.org:8080//job/charmrepo/47/
Test PASSed.

**Comment by frankban** [comment]
:shipit:

**Comment by frankban** [comment]
Thanks for the reviews!

**Comment by jujugui** [comment]
Status: merge request accepted. Url: http://ci.jujugui.org:8080/job/charmrepo-merge

**Pull request merged**